src/app/shared/ui/toggle/toggle.component.ts

```diff
@@ -0,0 +1,70 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-toggle',
+  standalone: true,
+  template: `
+    <div class="rabatCode">
```

Comment:
co to za nazwa klasy?

Reply:
musisz bardziej uważać przy kopiowaniu kodu z neta @jBachanek-02  xD

Reply:
to nie z neta XD